var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -26,34 +34,36 @@ class Ascent(Package):
     simulations."""
 
     homepage = "https://github.com/Alpine-DAV/ascent"
-    git      = "https://github.com/Alpine-DAV/ascent.git"
 
     maintainers = ['cyrush']
 
-    version('develop', branch='develop', submodules=True)
+    version('develop',
+            git='https://github.com/Alpine-DAV/ascent.git',
+            branch='develop',
+            submodules=True,
+            preferred=True)
```

Comment:
preferred isn't necessary if there is only one version.

Reply:
thanks, I removed another now broken version thats why its still there.  
Does it hurt to keep it?